packages/tasks/src/hardware.ts

```diff
@@ -0,0 +1,305 @@
+				tflops: 3.6,
+			},
+			"Xeon 2th Generation (Cascade Lake)": {
+				tflops: 3,
+			},
+			"Intel Core 13th Generation (i9)": {
+				tflops: 3.8,
+			},
+			"Intel Core 13th Generation (i7)": {
+				tflops: 3.6,
+			},
+			"Intel Core 13th Generation (i5)": {
+				tflops: 3,
+			},
+			"Intel Core 13th Generation (i3)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i9)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i7)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i5)": {
+				tflops: 2,
+			},
+			"Intel Core 12th Generation (i3)": {
+				tflops: 1,
+			},
+			"Intel Core 11th Generation (i9)": {
+				tflops: 1,
+			},
+			"Intel Core 11th Generation (i7)": {
+				tflops: 0.76,
+			},
+			"Intel Core 11th Generation (i5)": {
+				tflops: 0.76,
+			},
+			"Intel Core 11th Generation (i3)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i9)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i7)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i5)": {
+				tflops: 0.76,
+			},
+			"Intel Core 10th Generation (i3)": {
+				tflops: 0.76,
+			},
+		},
+		AMD: {
+			"EPYC 4th Generation (Genoa)": {
+				tflops: 4,
+			},
+			"EPYC 3th Generation (Milan)": {
+				tflops: 3,
+			},
+			"EPYC 2th Generation (Rome)": {
+				tflops: 2.9,
+			},
+			"EPYC 1st Generation (Naples)": {
+				tflops: 2.3,
+			},
+			"Ryzen Zen4 7000 (Ryzen 9)": {
+				tflops: 2.3,
+			},
+			"Ryzen Zen4 7000 (Ryzen 7)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen4 7000 (Ryzen 5)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen3 5000 (Ryzen 9)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen3 5000 (Ryzen 7)": {
+				tflops: 1.33,
+			},
+			"Ryzen Zen3 5000 (Ryzen 5)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Threadripper)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 9)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 7)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 5)": {
+				tflops: 0.72,
+			},
+			"Ryzen Zen 2  3000 (Ryzen 3)": {
+				tflops: 0.72,
+			},
+		},
+	},
+	"Apple Silicon": {
+		"-": {
+			"Apple M1": {
+				tflops: 2.6,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M1 Pro": {
+				tflops: 5.2,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M1 Max": {
+				tflops: 10.4,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M1 Ultra": {
+				tflops: 21,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M2": {
+				tflops: 3.6,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M2 Pro": {
+				tflops: 13.6,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M2 Max": {
+				tflops: 13.49,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M2 Ultra": {
+				tflops: 27.2,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M3": {
+				tflops: 2.84,
+				memory: [16, 24, 32, 64, 96, 128],
+			},
+			"Apple M3 Pro": {
+				tflops: 14,
+				memory: [16, 24, 32, 64, 96, 128],
```

Comment:
```suggestion
				memory: [18, 36],
```

Source: Apple purchase page